src/content/api/asset-types.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Asset Types
+description: Asset types supported for payment
+draft: true
+nav:
+  path: API
+  order: 10
+---
+
+The following table describes the Asset Types supported for payments.
+
+Asset types support live and test assets unless specified by supplying the correct suffix e.g. `centrapay.nzd.main` for live and `centrapay.nzd.test` for test.
+
+The Category column refers to the Centrapay asset type representation if
```

Comment:
```suggestion
The Category column refers to the Centrapay [Asset category](/api/assets/) representation if
```

Reply:
I have removed this line. It was a previous iteration of the table. I have now updated to have it mentioned in the legend instead.